cloudinit/stages.py

```diff
@@ -953,23 +956,38 @@ def should_run_on_boot_event():
 
 
 def read_runtime_config():
-    return util.read_conf(RUN_CLOUD_CONFIG)
```

Comment:
Cloud-init owns and creates this file as world-readable, so I don't believe we need the try/except handling around this.

Reply:
Removed, thanks.